rcswitch/user_doc.rst

```diff
@@ -0,0 +1,134 @@
+.. index:: Plugins; rcswitch (Brennenstuhl RCS 1000 N)
+.. index:: rcswitch
+
+================
+Plugins/rcswitch
```

Comment:
Der Titel ist falsch. Hier darf nur der Name des Plugins stehen (identisch zum Verzeichnisnamen des Plugins).

Reply:
erl. in PR für `rcs1000n`